be1-go/hub/standard_hub/mod.go

```diff
@@ -548,7 +552,8 @@ func (h *Hub) createLao(msg message.Message, laoCreate messagedata.LaoCreate,
 		return answer.NewInvalidMessageFieldError("failed to unmarshal public key of the sender: %v", err)
 	}
 
-	if !h.GetPubKeyOwner().Equal(senderPubKey) {
+	//TODO
+	if !h.GetPubKeyOwner().Equal(senderPubKey) && h.isCheckingOrganizerKey {
```

Comment:
I'd inverse them, so that the `!h.GetPubKeyOwner().Equal(senderPubKey)` is not even evaluated if we don't check the organizer